src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -412,6 +432,8 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
                 status: OperatorStatus.REGISTERED
             });
 
+            delegationManager.registerOperatorWithAVS(operator, signatureWithSaltAndExpiry);
```

Comment:
I wish we could skip this signature if the caller was the operator. Seems like unnecessary overhead, but I guess it makes sense in the context of untrusted AVSs...